docs/content/overview/product-telemetry.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Product Telemetry"
+description: "Learn how and why NGINX Ingress Controller collects product telemetry data."
+weight: 500
+toc: true
+---
+
+## Overview
+
+NGINX Ingress Controller collects product telemetry data to allow its developers to understand how it's deployed and configured by users.
+The goal is to use this data to triage development work and prioritize features and functionality that will provide the most value to our users.
+
+## About Product Telemetry
+
+Product telemetry is collected by default.
+Data is collected once every 24 hours, and is sent to a service managed by F5 over HTTPS.
+Personally identifiable information (PII) is **not** collected. 
+
+> [!NOTE]  
+> If you prefer to not have data collected, you can [opt-out](#opt-out) when installing NGINX Ingress Controller.
```

Comment:
```suggestion
{{< note >}}
If you would prefer to avoid sending any telemetry data, you can [opt-out](#opt-out) when installing NGINX Ingress Controller.
{{< /note >}}
```

This is going into Hugo, so we can use Hugo shortcodes. "prefer to not have" is a double negative and harder to read as a result. Even worse for non-native readers.